src/PensionsRegulatorApi/Data/OrganisationRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using PensionsRegulatorApi.Domain;
+
+namespace PensionsRegulatorApi.Data
+{
+    public interface OrganisationRepository
```

Comment:
IOrganisationRepository ?

Reply:
:) My thing for not putting magic "I"s in front of interfaces and then distinguishing the implementation by something meaningfull e.g. `SqlOrganisationRepository` otherwise feels a bit like Hungarian notation ;)

I don't mind changing though.

Reply:
Changed it.